cmd/rpc/web/wallet/components/account.jsx

```diff
@@ -115,6 +130,28 @@ export default function Accounts({ keygroup, account, validator }) {
     }
   }
 
+  // setActivePrivateKey() sets the active key to the newly added privte key if it is successfully imported
+  function setActivePrivateKey(nickname, closeModal) {
+    const resetState = () =>
+      setState({ ...stateRef.current, showSpinner: false, ...(closeModal && { [closeModal]: false }) });
+
+    retryWithDelay(
```

Comment:
should this be used in other places?

Reply:
I thought of doing it when creating a cnew account but a hash is returned, if that is not useful it can be added there too